test/abi/Inputs/macOS/arm64/stdlib/baseline-asserts

```diff
@@ -13493,6 +13493,7 @@ __swift_debug_protocolConformanceStatePointer
 __swift_debug_verifyTypeLayoutAttribute
 __swift_disableExclusivityChecking
 __swift_dumpObjectsRuntimeFunctionPointers
+__swift_enableSwizzlingOfAllocationAndRefCountingFunctions_forInstrumentsOnly
```

Comment:
Can you put this in the exception file instead of the baseline? `test/abi/macOS/*/stdlib.swift`

Reply:
Oh wow I’ve been away too long, I should have remembered that. Thanks for catching it!